### PR TITLE
Moving py smb to runtime

### PR DESCRIPTION
**Pull request by Mzack9999**
This PR creates smb python server file at runtime in /tmp

### CODE REVIEW

File: pkg/server/smb_server.go [hunk 1]
```diff
@@ -46,16 +46,26 @@ func (h *SMBServer) ListenAndServe(smbAlive chan bool) error {
...
 	}
-	h.tmpFile = tmpFile.Name()
-	tmpFile.Close()
-	// execute smb_server.py - only works with ./interactsh-server
-	cmdLine := fmt.Sprintf("python3 smb_server.py %s %d", h.tmpFile, h.options.SmbPort)
-	args := strings.Fields(cmdLine)
-	h.cmd = exec.Command(args[0], args[1:]...)
+	pyFileName += ".py"
+
+	if err := os.WriteFile(pyFileName, []byte(pySmbServer), os.ModePerm); err != nil {
```
Comment (dogancanbakir) [inline pkg/server/smb_server.go:62]: We can consider using `permissionutil.TempFilePermission`.

File: pkg/server/smb_server.go [hunk 2]
```diff
@@ -46,16 +46,26 @@ func (h *SMBServer) ListenAndServe(smbAlive chan bool) error {
...
+		return err
+	}
+
+	// args := strings.Fields(cmdLine)
```
Comment (dogancanbakir) [inline pkg/server/smb_server.go:66]: If it's unnecessary, we can remove the commented-out code.

**Review by dogancanbakir — APPROVED** [review]
LGTM - left comments for minor changes

File: pkg/server/smb_server.go (hunk 1 above)
Comment (Mzack9999) [reply pkg/server/smb_server.go:62]: opted for this as we need also `+x`

File: pkg/server/smb_server.go (hunk 2 above)
Comment (Mzack9999) [reply pkg/server/smb_server.go:66]: done!

**Pull request merged**